README.md

```diff
@@ -26,7 +26,10 @@ check out and contribute to this repository.
 Important links:
 
 - [Workshops](workshops/README.md) :: Workshops for use in clubs
-- https://slack.hackclub.io :: Join our [Slack](https://slack.com/) community
+- https://slack.hackclub.io :: Join
+  [our Slack](https://starthackclub.slack.com/) community
```

Comment:
Ah, but the Slackin is already linked on the left, right? (the `https://slack.hackclub.io :: ` bit).

I think it's worthwhile to have a link to Slack itself in there for people who don't know what Slack is.